src/CodeGeneratorGo.cs

```diff
@@ -46,11 +46,9 @@ public override async Task Generate(CodeModel cm)
             var folder = Path.GetFullPath(Settings.Instance.Host.GetValue<string>("output-folder").Result).Replace('\\', '/');
             // check if the namespace contains illegal characters
             var ns = Settings.Instance.Host.GetValue<string>("namespace").Result;
-            var r = new Regex(@"^[a-z][a-z0-9_]*[a-z0-9]?$");
-            // skip this check if namespace is not specified
-            if (!string.IsNullOrEmpty(ns) && !r.IsMatch(ns))
+            if (Settings.Instance.Host.GetValue<bool>("namespace-chk").Result)

```

Comment:
We're putting it behind a flag because the failure is a problem for swagger authors (similar to the preview thing)?

Reply:
well, I am not sure whether it is the code generator's obligation to verify the generated code could work (similar problem happens in the `nextLink`s), but this namespace works as a package name, wrong namespace will certainly cause the generated code not working.
Empty namespace or wrong namespace is truly a problem that the swagger author should resolve (usually something wrong in the readme).

Reply:
Agreed.  I think this was another change I had made in the code generator when the POR was to work towards auto-generation and release of modules.

Reply:
I am sorry but what does "POR" stand for here?

Reply:
Plan Of Record